### ISSUE TITLE
Previous addresses

### ISSUE THREAD

**Issue by usmarine2141** [Coinomi/coinomi-android#193]
Are our previous addresses able to still be sent funds?

**Comment by tzarebczan** [comment]
Yes, absolutely. Every address in your wallet is yours and privately controlled.

**Comment by usmarine2141** [comment]
Awesome. For privacy concerns are there ways to track those once or twice used addresses? 

Is there a way to toss that wallet address like trash it

**Comment by tzarebczan** [comment]
If you get a new address for each tx, you are guaranteed to use each address only once. But then your coins are stored there.. There's no way to delete the tx or address after it happened, even without any balance.

**Comment by usmarine2141** [comment]
Okay. But there is not a finite way to track that wallet to a particular
person without a huge amount of work right?

Only asking not for nefarious reasons but for family law legalities

**Comment by tzarebczan** [comment]
A good explanation here https://bitcoin.stackexchange.com/questions/30808/hd-wallets-and-address-reuse
